master/reference/calicoctl/resources/ippool.md

```diff
@@ -51,3 +51,6 @@ is in an IP Pool that has IPIP enabled.  In addition, if the `mode` is set to `c
 Calico will only route using IP in IP if the IP address of the destination node is in a different
```

Comment:
This is already covered in the IP-in-IP guide - so it's duplicate info - I think it's probably worth including unless it's inaccurate.